src/main/pages/setup-kubernetes/kubernetes-config.md

```diff
@@ -81,4 +81,11 @@ pods almost instantly and significantly decrease the time required for stopping
 
 If `terminationGracePeriodSeconds` have been explicitly set in Kubernetes / OpenShift recipe it will not be overridden by the environment variable.
 
+## Che Workspace Unrecoverable Events
+
+By default, if one of the following Kubernetes / OpenShift events (`Failed Mount` / `Failed Scheduling` / `Failed`) occurs during a startup, workspace will be immediately terminated before timeout.
```

Comment:
Is that intentional or have you forgot `to pull image` in the last event?

Reply:
yeah, I thought any `Failed` event should trigger startup failure, but I can rework if you think it is relevant